src/components/side_panel/chart/chart_with_axis/design_panel.xml

```diff
@@ -98,8 +98,20 @@
                     <option value="logarithmic" t-att-selected="trendType === 'logarithmic'">
                       Logarithmic
                     </option>
+                    <option value="movingAverage" t-att-selected="trendType === 'movingAverage'">
+                      Moving average
```

Comment:
should we also make it clear for the user ? That we he could even google it if the user wants to better understand what it is/does.
```suggestion
                      Trailing moving average
```